templates/search.html.twig

```diff
@@ -1,5 +1,14 @@
 {% extends 'base.html.twig' %}
 
+{% block title %}
+    <h1>{{ msg('svgtranslate') }}</h1>
+    <p>{{ msg('info') }}</p>
+    <p>
+        {{ msg('beta') }}
```

Comment:
Do we need a space after this message and before the link?

Reply:
There is one (by virtue of the line after this being indented). We don't have [spaceless](https://twig.symfony.com/doc/2.x/templates.html#whitespace-control) turned on in this bit.

Reply:
Got ya. HTML/Twig magic.